tests/integration/test_charm.py

```diff
@@ -103,6 +112,88 @@ async def test_create_profile_action(lightkube_client, ops_test):
     validate_profile_namespace(lightkube_client, profile_name)
     validate_profile_owner(lightkube_client, namespace, profile_name, auth_username)
     validate_profile_resource_quota(lightkube_client, namespace, profile_name, expected_quota)
+    validate_namespace_poddefaults(lightkube_client, profile_name)
+
+
+async def test_initialise_profile_action(lightkube_client, profile, ops_test):
+    """Test profile initialisation action."""
+    profile_name = profile
+
+    action = (
+        await ops_test.model.applications[CHARM_NAME]
+        .units[0]
+        .run_action(
+            "initialise-profile",
+            profilename=profile_name,
+        )
+    )
+    await action.wait()
+
+    validate_namespace_poddefaults(lightkube_client, profile_name)
+
+
+async def test_initialise_profile_action_copy_seldon_secret(lightkube_client, profile, ops_test):
+    """Test initialise profile action to copy seldon secret when secret is in kubeflow namespace."""
+    # create kubeflow namespace
+    namespace_name = "kubeflow"
+    namespace_metadata = ObjectMeta(name=namespace_name)
+    namespace = Namespace(metadata=namespace_metadata)
+    lightkube_client.create(namespace, namespace_name)
+
+    # create seldon secret in kubeflow namespace
```

Comment:
another approach to this would be to deploy mlflow then copy the secret created on install of mlflow-server charm

Reply:
IMO this is sufficient.